Foundation/include/Poco/NumberFormatter.h

```diff
@@ -43,6 +43,17 @@ class Foundation_API NumberFormatter
 		FMT_ON_OFF
 	};
 
+	enum class Options
+		/// Option to control the format of the generated string.
+	{
+		OPT_DEFAULT = 0,
+			/// formatHex defaults: No 0x prefix, uppercase hexadecimal values
+		OPT_PREFIX = 1 << 0,
+			/// formatHex: Prepend prefix 0x
+		OPT_LOWERCASE = 1 << 1
```

Comment:
I still think if it is good to have OPT_ prefix. At least I do not use those with enum classes. Those where totally different when just enum where used.

```
Options::OPT_DEFAULT;
Options::DEFAULT;

Options::OPT_PREFIX;
Options::PREFIX;

Options::OPT_LOWERCASE;
Options::LOWERCASE;
```

And this is nitpicking and I leave it up to you to choose which you prefer. My vote go still without OPT_. Everything looks very good. Nice job.

Reply:
Renamed. Now it is the time to do it right. It will not be changed soon again.